### PR TITLE
whitelist core

### PR DESCRIPTION
**Pull request by ericvergnaud**
## Changes
whitelist core

### Linked issues
Progresses #1901 

### Functionality
None

### Tests
- [x] manually tested

### CODE REVIEW

File: src/databricks/labs/ucx/source_code/known.json [hunk 1]
```diff
@@ -1278,6 +1278,11 @@
   "contourpy": {
     "contourpy": []
   },
+  "core": {
```
Comment (JCZuurmond) [inline src/databricks/labs/ucx/source_code/known.json:1281]: What kind of package is this?

**Review by JCZuurmond — COMMENTED** [review]
See comment

File: src/databricks/labs/ucx/source_code/known.json (hunk 1 above)
Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:1281]: I believe it's this one: https://pypi.org/project/python-core/

Comment (JCZuurmond) [reply src/databricks/labs/ucx/source_code/known.json:1281]: can you add it again as a comment? It is a weird package, no info at all

Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:1281]: As already mentioned, json does not support comments

Comment (JCZuurmond) [reply src/databricks/labs/ucx/source_code/known.json:1281]: ah missed that, that is unfortunate

Comment (JCZuurmond) [reply src/databricks/labs/ucx/source_code/known.json:1281]: https://pypi.org/project/python-core/ does not look like the right package, contents is as follows:

Screenshot 2024-07-11 at 15 12 55

Comment (ericvergnaud) [reply src/databricks/labs/ucx/source_code/known.json:1281]: Actually it comes from https://pypi.org/project/dbl-sat-sdk/0.1.34/

Comment (nfx) [reply src/databricks/labs/ucx/source_code/known.json:1281]: this is noise package

**Pull request closed without merging**